src/dwi/fmls.h

```diff
@@ -160,6 +160,9 @@ namespace MR
       };
 
 
+      bool order_by_fd (const FOD_lobe&, const FOD_lobe&);
```

Comment:
Wouldn't it be worth declaring this function `inline` and define it fully at this point? It's unlikely to make that much difference to your use case given that the function is defined in the same cpp file as it's going to be used, so the compiler will likely inline the code anyway (even though you've basically told it not to). But it might impact on the code if used in other cpp files. 

On the other hand, if you don't intend for that function to be used outside of `src/dwi/fmls.cpp`, then I'd declare and define it `inline` within an anonumous namespace in `src/dw/fmls.cpp` - no need to expose internals of your implementation if you have no intention to allow other parts of the code to make use of it...

Reply:
39ab87b?